### PR TITLE
[DO NOT MERGE !!!] SLB changes for new backend pool addresses format.

### PR DESCRIPTION
**Pull request by ninzavivek**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/loadBalancer.json [hunk 1]
```diff
@@ -413,6 +413,9 @@
         "x-ms-examples": {
           "LoadBalancerBackendAddressPoolList": {
             "$ref": "./examples/LoadBalancerBackendAddressPoolList.json"
+          },
+          "LBBackendAddressPoolListWithBackendAddressesPoolType": {
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/loadBalancer.json]: Please make this a human-readable description

File: specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/loadBalancer.json [hunk 2]
```diff
@@ -1347,6 +1427,34 @@
...
+          "$ref": "./network.json#/definitions/SubResource",
+          "description": "Reference to an existing virtual network."
+        },
+        "ipAddress": {
+          "type": "string",
+          "description": "IP Address belonging to the virtual network referenced above.",
+          "x-ms-azure-resource": false
+        }
+      },
+      "description": "Properties of the loadbalancer backend addresses."
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/loadBalancer.json]:
```suggestion
      "description": "Properties of the load balancer backend addresses."
```

File: specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/loadBalancer.json [hunk 3]
```diff
@@ -1347,6 +1427,34 @@
...
     },
+    "LoadBalancerBackendAddressPropertiesFormat": {
+      "properties": {
+        "virtualNetwork": {
+          "$ref": "./network.json#/definitions/SubResource",
+          "description": "Reference to an existing virtual network."
+        },
+        "ipAddress": {
+          "type": "string",
+          "description": "IP Address belonging to the virtual network referenced above.",
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/loadBalancer.json]:
```suggestion
          "description": "IP Address belonging to the referenced virtual network.",
```

File: specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/loadBalancer.json [hunk 4]
```diff
@@ -1347,6 +1427,34 @@
...
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/LoadBalancerBackendAddressPropertiesFormat",
+          "description": "Properties of load balancer backend address pool."
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the backend address."
+        }
+      },
+      "description": "Loadbalancer backend addresses."
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/loadBalancer.json]:
```suggestion
      "description": "Load balancer backend addresses."
```

**Review by anton-evseev — CHANGES_REQUESTED** [review]
LGTM except a few nitpicks

**Comment by lirenhe** [comment]
@ninzavivek, I saw this is still in draft mode, please let us know when it is ready for SDK/ARM review.

**Comment by number213** [comment]
Closed in favor of https://github.com/Azure/azure-rest-api-specs/pull/8993

**Pull request closed without merging**